pkg/webhook/workloadspread/validating/workloadspread_validation.go

```diff
@@ -137,10 +138,14 @@ func validateWorkloadSpreadSpec(obj *appsv1alpha1.WorkloadSpread, fldPath *field
 				spec.ScheduleStrategy.Adaptive.RescheduleCriticalSeconds, "the last subset must be not specified when using adaptive scheduleStrategy"))
 		}
 
+		allowedMaxSeconds := int32(math.MaxInt32)
+		if len(spec.Subsets) > 1 {
+			allowedMaxSeconds = int32(300-5) / int32(len(spec.Subsets)-1)
```

Comment:
what is the magic number means here (e.g. 300 and 5), consider replace them with meaningful const variables

Reply:
done